README.md

```diff
@@ -1,46 +1,35 @@
-# Sway Visual Studio Code Plugin
+<picture>
+   <source media="(prefers-color-scheme: light)" srcset="./images/light/sway.png">
+   <source media="(prefers-color-scheme: dark)" srcset="./images/dark/sway.png">
+   <img alt="Sway VSCode Plugin" src="./images/dark/sway.png">
+</picture>
 
 [![Visual Studio Marketplace Version](https://img.shields.io/visual-studio-marketplace/v/FuelLabs.sway-vscode-plugin)](https://marketplace.visualstudio.com/items?itemName=FuelLabs.sway-vscode-plugin)
 [![discord](https://img.shields.io/badge/chat%20on-discord-orange?&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2)](https://discord.gg/xfpK4Pe)
 
-## When does the plugin get activated?
+This extension provides support for the Sway smart contract programming language.
 
-Currently it gets activated once you open a file with `.sw` extension.
+## Features
 
-## Testing
+- go to definition, implementation, type definition
+- symbol renaming
```

Comment:
We don't yet support this capability either. We could add this to the coming soon section below though.